setup.py

```diff
@@ -21,7 +21,15 @@
     readme = f.read().split('--------------------')[-1]
 
 with open('requirements.txt') as f:
-    reqs = f.read()
+    reqs = []
+    for line in f:
+        line = line.strip()
+        if '>' in line:
+            reqs.append(line.split('>')[0])
+        elif '=' in line:
```

Comment:
we won't be expecting '~=' in , '<=' equirements.txt in the future?

Reply:
only this nit, other LGTM.

Reply:
Does ~= work?

Reply:
Maybe I can just use line.split('=').rstrip('><=~')...

Reply:
yeah that could work.